README.md

```diff
@@ -102,6 +102,23 @@ tokenize(command.toLowerCase())
     )
 ```
 
+### Binding (Monad Comprehension)
+
+The `binding` keyword is provided to allow an imperative-like style of working with the result type:
+```kotlin
```

Comment:
i thought it best to let the code do the talking here, but now that i read over it, maybe it would make sense to have a section before talking about its inspiration that goes into a little more detail on whats happening?

maybe just a copy paste of the description inside `Binding.kt`?

Reply:
went ahead and added that in, also made a slight change to the current description of `binding`, let me know what you think.